streamer/parser/metadata.md

```diff
@@ -6,6 +6,17 @@ While the default context is very flexible in the way each metadata can be repre
 
 Related Repository: [Readium Web Publication Manifest](https://github.com/readium/webpub-manifest)
 
+## Localized Strings
+
+In many cases, the default context supports alternate representations of the same string in different scripts and languages by means of JSON-LD language maps.
+To fill such a map from an EPUB metadata element, proceed as follows:
+
+* Determine the language used in the content of the carrying element as defined in [the XML specification](https://www.w3.org/TR/xml/#sec-lang-tag),
+  i.e. check whether the carrying element has or inherits an `xml:lang` attribute. Otherwise, fallback to the primary language of the publication.
+* In the EPUB 3.x case, check if the element is refined by some `meta` elements that have or inherit an `xml:lang` attribute and whose property is `alternate-script`.
+  For each one, add to the map the corresponding language associated with the content of the `meta` element.
+
+
```

Comment:
Question: should this be updated now that the decision to use `und` has been made?

Reply:
How would you update this? Specifying that as a last fallback `null` or `und` is used? (this is equivalent in the Kotlin code)
Or are you talking about removing 

> Otherwise, fallback to the primary language of the publication.

Unfortunately, as far as I know, no decision has been taken about that. Do you have an opinion? (see https://github.com/readium/webpub-manifest/issues/52#issuecomment-601625302)

Reply:
Ah so during the review of the accessibility parsing doc, @llemeurfr mentioned the use of `und` here for `sortAs`: https://github.com/readium/architecture/pull/135#discussion_r417879475

So I assumed that resolution was made during on of the calls I missed and that this PR should perhaps be updated accordingly. In any case, there seems to be a discrepancy between PRs and issues right now.

Reply:
Okay, I drafted an update to take into account recent decisions.

Reply:
Thanks!